app/controllers/plans_controller.rb

```diff
@@ -32,9 +32,11 @@ def update
     @plan = Plan.find(params[:id])
 
     if @plan.update(plan_params)
-      redirect_to @plan
+      flash[:success] = 'Plan was successfully updated.'
+      redirect_to edit_plan_path
     else
-      render 'edit'
+      flash[:danger] = 'Plan was not created.'
```

Comment:
As you're in the `#update` method, maybe you should change the string to `'Plan was not updated'`.